src/classes/frSetupController.cls

```diff
@@ -69,6 +87,19 @@ public with sharing class frSetupController {
 	public List<SelectOption> donorSFOptions {get; set;}
 	public List<SelectOption> donorFROptions {get; set;}
 	public List<frMapping__c> donorMappings {get; set;}
+
+	public List<SelectOption> accSFOptions {get; set;}
```

Comment:
I don't think we should be duplicating the selectoption and frMapping variables here.  The above mapping variables aren't specific to Opportunity/Contact, we should just change what options are in there.

Reply:
This was to make the code a bit easier to read/debug. This can be removed if needed but wont affect functionality that im aware of.

Reply:
yes, please remove.  

I think having added variables that are duplicating things the other variables are doing makes it a bit more confusing and harder to manage (especially because there are not comments that explain why there are so many options that would not be used in certain environments) rather than having each Funraise object having a single select list of options, and the object used to populate those objects is what changes per environment (NPSP vs NPC vs neither).   

The gift commitment options are the only ones that I think would need to be added since there isn't currently mappable options for subscriptions/commitments